src/OrchardCore.Modules/OrchardCore.Lists/Services/ContainerService.cs

```diff
@@ -146,7 +153,7 @@ public async Task SetInitialOrder(string contentType)
             }
         }
 
-        public async Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string contentItemId, bool enableOrdering, PagerSlim pager, bool publishedOnly)
+        public async Task<IEnumerable<ContentItem>> QueryContainedItemsAsync(string contentItemId, bool enableOrdering, PagerSlim pager, bool publishedOnly, ListPartFilter listPartFilter)
```

Comment:
Szerintem a filter és a `publishedOnly` összevonható, utóbbira nincs szükség, kiváltható.

Reply:
Összevonást azt hogy képzeljem el? Módosítsam az ide tartozó interface-t?

Reply:
A `publishedOnly` nem kell, helyette ezt megoldhatod az új class-szal is. És igen, ehhez módosítani kell az interace-t is.

Reply:
Az új osztálynál gondolom a szűréshez létrehozott model osztályra gondolsz, amit itt utolsó paraméterként adok át. Ennek van ugye a Status property-je ami enum, és abban van a published érték (ami 2). Tehát ha átírom az interface-t hogy bool helyett ezt fogadja, akkor a kód többi részét is átírom, ahol a bool változóval dolgozott?

Reply:
Persze.